src/frontend/staff-portal/src/app/components/staff-workbench/ticket-info/ticket-info.component.ts

```diff
@@ -183,9 +183,15 @@ export class TicketInfoComponent implements OnInit {
     this.form.get('contactLawFirmNm').updateValueAndValidity();
     this.form.get('contactSurnameNm').updateValueAndValidity();
     this.form.get('contactGivenNames').updateValueAndValidity();
+    this.onFieldChange();
     this.form.updateValueAndValidity();
   }
 
+  public onFieldChange() {
+    const ctryId: number = this.form.get('addressCountryId').value;
+    this.onCountryChange(ctryId);
```

Comment:
Can you try not calling onFieldChange() and call onCountryChange in getDispute() since those fields seems unrelated